### DISCUSSION TITLE
[Feature request] display stack tree in PR body

### DISCUSSION THREAD

**Discussion by kwonoj** [aviator-co/av#493] [Ideas]
`av` works quite well for me for the initial workflow. `av tree` is also neat.

But in the pull request, the reviewers seems having confusion about the sequence of the pull request for their review.

Is it possible to embed or update a stack tree in the PR body, similar to other tools? I think this feature could greatly assist reviewers in understanding the order of the pull requests.
<img width="552" alt="image" src="https://github.com/user-attachments/assets/e63f7e69-f46a-4b70-a5b3-d4a65b8905ea" />

**Comment by jainankit** [comment]
You should be able to do this with config:
https://docs.aviator.co/aviator-cli/configuration

```
pullRequest:
  writeStack: true
```

**Reply by kwonoj** [reply]
Wow, thanks. 🤦 that I didn't read docs fully.
